src/Intercom/Data/ConversationPart.cs

```diff
@@ -18,7 +14,7 @@ public class ConversationPart : Model
         public long created_at { get; set; }
         public long updated_at { get; set; }
         public long notified_at { get; set; }
-        public object assigned_to { get; set; }
+        public string assigned_to { get; set; }
```

Comment:
Why is this a string now?

Reply:
Seems to have been changed when we added group conversations:

![image](https://user-images.githubusercontent.com/11810153/36105983-f2025f5e-100d-11e8-906f-2de224d2ecd3.png)

In [here](https://developers.intercom.com/v2.0/reference#conversation-part-model).